panels/models/attraction.py

```
@@ -488,24 +633,37 @@ def start_time_label(self):
         return 'unknown start time'
 
     @property
-    def required_checkin_time(self):
-        required_checkin = self.feature.attraction.required_checkin
-        if required_checkin < 0:
-            return self.end_time
+    def checkin_time(self):
+        advance_checkin = self.attraction.advance_checkin
+        if advance_checkin < 0:
+            return self.start_time
         else:
-            return self.start_time - timedelta(seconds=required_checkin)
+            return self.start_time - timedelta(seconds=advance_checkin)
+
+    @property
+    def checkin_time_local(self):
+        return self.checkin_time.astimezone(c.EVENT_TIMEZONE)
```

Do we no longer have a suffix property which automatically does this for all UTCDateTime column fields on ``MagModel``?

---

Hah! I didn't even know that existed! I will remove these properties. I have a real love/hate relationship with `@suffix_property`. On the one hand, it's super convenient. On the other hand, our implementation means that any errors in a property are completely swallowed and hidden. See the following issues for more discussion:
* https://github.com/magfest/ubersystem/issues/1942
* https://github.com/magfest/ubersystem/issues/2842